doc/quickstart/ETF_quickstart.rst

```diff
@@ -0,0 +1,258 @@
+
+A Test Report can be opened again by clicking on Open report or can be downloaded as HTML file by clicking on the Download button.
+
+The log file of the test run can be inspected with the Open log button. By clicking on Delete report button, the report will be deleted permanently.
+
+
+Inspect test reports
+============
+
+The top of a Test Report shows general information including the overall test result Status, the start time, the duration and a statistical table, which summarizes the status of all tests on several levels.
+
+.. image:: /images/projects/ETF/inspect-test-reports-1.png
+    :scale: 70 %
+
+The Test Reports are interactive. The Show switch can be used to filter Only failed or Only manual tests. All deactivates the filter.
+
+The Level of detail switch is used to show additional technical information in the reports.
+
+.. image:: /images/projects/ETF/inspect-test-reports-2.png
+    :scale: 70 %
+
+The test results are summarized hierarchically in a report. At the top level there are the Test Suites.
+
+By clicking on one test suite a description and all lower level tests in that test suite are shown. Failures in a test suite can be immediately recognized by the red color. The number of failed tests is shown in the top-right corner.
+
+.. image:: /images/projects/ETF/inspect-test-reports-3.png
+    :scale: 70 %
+
+The green color indicates a passed test. Passed tests which require additional manual test steps that could not be automated are colored orange. The orange color may also indicate a test that has been skipped because it depends on another test that has failed. The exact status can be found below the description.
+
+The number of levels depends on the tested Test Object. If service tests have been executed the hierarchy is as follows:
+
+        * Executable Test Suites
+
+        * Test Modules (bundles Test Cases)
+
+        * Test Cases (bundles Test Steps)
+
+        * Test Steps (interactions with the service, bundles Test Assertions)
+
+        * Test Assertions (atomar tests)
+
+In a file-based test, Test Modules and Test Steps do not exist and are not shown in the report.
+
+Each test provides a description on how aspects are tested and lists the requirements. The test may possess a link to an abstract test suite, from which the test has been derived (Source).
+
+.. image:: /images/projects/ETF/inspect-test-reports-4.png
+    :scale: 70 %
+
+Assertions stand for atomic test queries on the lowest level. Failed, red colored assertions display error messages in the Messages section.
+
+.. image:: /images/projects/ETF/inspect-test-reports-5.png
+    :scale: 70 %
+
+Helpful information may also be found on the next higher level, like for instance the response from a service on the Test Step level (note the Open saved response link in the report).
+
+.. image:: /images/projects/ETF/inspect-test-reports-6.png
+    :scale: 70 %
+
+
```

Comment:
The quickstarts normally end with a What's next? caption. Maybe you can add something like that
https://github.com/OSGeo/OSGeoLive-doc/blob/master/doc/quickstart/R_quickstart.rst

Reply:
Resources and Whats next added